### ISSUE TITLE
MSRV 1.54.0

### ISSUE THREAD

**Issue by taqtiqa-mark** [zombiezen/redo-rs#17]
Document a MSRV of 1.54.0 (x86_64).  

Unlikely to need to change this for different platforms (macos, embedded/arm) but that may be a possibility?

Arrived at via `cargo-msrv`:

````bash
$ cargo msrv --bisect
Fetching index
Determining the Minimum Supported Rust Version (MSRV) for toolchain x86_64-unknown-linux-gnu
Using check command cargo check --all
   Finished The MSRV is: 1.54.0
````

**Comment by taqtiqa-mark** [comment]
Closed when PR #20 lands

**Comment by taqtiqa-mark** [comment]
Closing as prior feedback indicates stable is the only target.
